README.md

```diff
@@ -60,21 +44,25 @@ source /path-to-where-spack-was-cloned/spack/share/spack/setup-env.sh
 
 ### Installing Dependencies
 
-Currently, most of the dependencies are listed in `spack.yaml` and can be installed via Spack.`gcc` and `g++` will be needed to build ChronoLog.
+Currently, most of the dependencies are listed in `spack.yaml` and can be installed via Spack. `gcc` and `g++` will be
+needed to build ChronoLog.
+
+A Spack environment needs to be created and activated using the following commands. When the environment is activated, a
+shell prompt `[ChronoLog]` will pop up.
```

Comment:
to me, the `[ChronoLog]` prompt in Spack v0.21.2 only works for bash, not zsh. zsh does not recognize the color syntax of bash, like \[\033[0;92m\][ChronoLog]\[\033[0m\] will be parsed as it is not [ChronoLog] in green. and for some terminal, nothing pops up after the activation.
I would recommend users to use `spack env status` to confirm if they are using the ChronoLog Spack env. if it works, the output should be something like `==> Using spack.yaml in current directory: /path/to/ChronoLog`. otherwise, it will be `==> No active environment`. You can add it as the last command and add the example output to the next section.